website/project/views/contributor.py

```diff
@@ -135,7 +136,18 @@ def get_contributors_from_parent(auth, node, **kwargs):
         if contrib._id not in node.visible_contributor_ids
     ]
 
-    return {'contributors': contribs}
+    size = int(request.args.get('size', len(contribs)))
+    page = int(request.args.get('page', 0))
+
+    contribs, pages = paginate(contribs,
+                               len(contribs),
+                               page,
+                               size)
+    contribs = list(contribs)
+
+    return {'users': contribs,
+            'page': page,
+            'pages': pages}
```

Comment:
This code block is repeated 3 times. It is worthy to write a help function to reduce duplicate code.